configs/walk study.nrel-op.json

```diff
@@ -0,0 +1,85 @@
+{
+  "url_abbreviation": "walk study",
```

Comment:
```suggestion
  "url_abbreviation": "portland-walk",
```

Reply:
@smkothuri do you have any thoughts on the recommended change? There might be other walk studies running in parallel, so it would be best if we could have your program or university name embedded in the URL